.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on: 
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ['main']
+
+# Sets the GITHUB_TOKEN permissions to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
```

Comment:
Is the concurrency setting needed? can we get away with not including it? In general, If something can be implied, or if applied defaults do the job, it would help us keep our action less verbose and maybe this is something we would like

Reply:
Needed if we don't want multiple builds to be triggered with each push, if they're done quickly.